### PR TITLE
EDU-1419 | Temporal Cloud guide | Introduction & get started content updates

### PR DESCRIPTION
**Pull request by flossypurse**
**Preview:** https://temporal-documentation-git-edu-1419.preview.thundergun.io/

New and updated content at:
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/production-deployment
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/introduction
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/overview
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/service-availability
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/limits
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/sla
- https://temporal-documentation-git-edu-1419.preview.thundergun.io/cloud/get-started

### CODE REVIEW

**Comment by axfelix** [comment]
First impression is that, because Temporal Cloud is currently not a top-level side nav element, this feels like a lot of nesting, and it's not totally clear to me why some of this stuff needs to be under "Introduction" as opposed to directly under Cloud:

image

Content looks great, but this is an awful lot of side nav.

**Comment by axfelix** [comment]
- not sure the topology diagrams on the production deployments landing page need to be collapsed by default, there isn't too much on that page otherwise
- "Become familiar with Temporal Cloud" feels like a placeholder -- do we even need a heading there at all? I still feel like we don't have a great sense of what belongs in "Introduction" here, that additional nav level is weird. 
- SLA, availability, etc., all look great, but people would probably need to consult these as a reference at some point -- I don't know if they're "Introduction" content.

**Comment by flossypurse** [comment]
> Content looks great, but this is an awful lot of side nav.
@axfelix - Ya, there will be tons of it. Side nav everywhere.

File: docs-src/cloud/configuration.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+description: Learn more about Temporal Cloud defaults, limits, and configurable settings.
+tags:
+  - temporal cloud
+  - defaults
+  - limits
+  - configuration
+  - settings
+---
+
+Temporal Cloud has many operational features, many of which that support customization.
```
Comment (dandavison) [inline docs-src/cloud/configuration.md:14]:
```suggestion
Temporal Cloud has many operational features, and many of them can be customized.
```

File: docs-src/production-deployment/landing.md [hunk 2]
```diff
@@ -7,24 +7,38 @@ tags:
   - production deployment
 ---
 
-**Deploying to production can be as simple as updating your connection endpoints to point at Temporal Cloud.**
+**Ready to evelate your durable application into production?**
```
Comment (axfelix) [inline docs-src/production-deployment/landing.md:10]: Love that the word "simple" has been removed, but this should probably be "elevate" :)

File: docs-src/cloud/available-regions.md [hunk 3]
```diff
@@ -9,7 +9,17 @@ tags:
...
 ---
 
-Temporal Cloud currently runs in the following regions in Amazon Web Services (AWS):
+**Where is Temporal Cloud available?**
+
+Developers and applications can access Temporal Cloud from any location with internet connectivity, irrespective of where the Temporal Cloud resources (Namespaces) are located.
+
+Temporal Cloud is compatible with applications deployed in various cloud environments or data centers.
+
+_**Your Workers and Client code aren't required to be hosted on AWS.**_
```
Comment (axfelix) [inline docs-src/cloud/available-regions.md:18]: This should probably get reordered to below the mention of AWS regions, right now it feels a bit out of pocket

**Pull request merged**